source/matplot/axes_objects/vectors.cpp

```diff
@@ -97,13 +142,24 @@ namespace matplot {
                     ss << "  " << 0.0;
                 }
 
+                if (normalize_) {
+                    double mag = sqrt(u_value * u_value + v_value * v_value +
```

Comment:
If I'm not wrong, I think some compilers require `std::sqrt` instead of just `sqrt`. I think the standard says these functions, when included from `<cmath>`, should be in the `std` namespace, but the calls outside `std` often end up working because some implementations just `#include <math.h>` inside `<cmath>` and they don't care to change the visibility of the original C functions.

Reply:
Good to know, I will make the change:
```suggestion
                    double mag = std::sqrt(u_value * u_value + v_value * v_value +
```